tests/lax_numpy_test.py

```diff
@@ -4670,6 +4670,33 @@ def testMgrid(self):
                         atol=atol,
                         rtol=rtol)
 
+  def testOgrid(self):
+    assertAllEqual = partial(self.assertAllClose, atol=0, rtol=0)
```

Comment:
I believe `self.assertArraysEqual` will do what you want here.

Reply:
Thanks for the review!

I tried to use `self.assertArraysEqual` instead of custom `assertAllEqual`.
However, in the case of `self.assertArraysEqual(np.ogrid[0:5, 1:3], jnp.ogrid[0:5, 1:3])`, it raised an error, which is `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`.
Is there a good work around for this exception? Thanks :)

Reply:
Oh, I see: the issue is you're not comparing arrays, you're comparing tuples of arrays. I think you'll need a custom comparator... the `assertAllClose` thing happens to work, but was not designed for this situation, so it's probably best not to rely on an accident of its implementation. Maybe define a custom function like `assertTuplesOfArraysEqual` and use that?

Reply:
Thanks. Now I see the problem!

I defined `assertListOfArraysEqual` (I believe ogrid returns list of arrays. Please correct me if I am wrong :) and use assertArraysEqual instead.

PTAL!